### PR TITLE
Nav/Subnav broken UI with PageSkins for leftCol and wide breakpoints.

### PR DESCRIPTION
**Pull request by deedeeh**
## What does this change?

This PR has major changes to the Navigation and subnav UI with PageSkins.  I had to match the `leftCol` and `wide` breakpoints  to `desktop` to have the correct UI.

## Why?

Better user experience and a UI that works as intended. 

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| image |  image |
| image | image |

[before]: https://example.com/before.png
[after]: https://example.com/after.png

### CODE REVIEW

File: dotcom-rendering/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "files.watcherExclude": {
```
Comment (domlander) [inline dotcom-rendering/.vscode/settings.json:2]: Can this be moved up a level, above `dotcom-rendering`, to your `.vscode/settings.json` file? If you don't have one, you can create one using these two files: 
- `.vscode/settings.json.recommended`
- `.vscode/settings.json.required`

Up a level, the  `.vscode/settings.json` file is ignored by git

File: dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx [hunk 2]
```diff
@@ -200,14 +200,30 @@ const columnStyle = css`
...
+		width: 134px;
+
+		:first-of-type {
+			width: 123px;
+		}
+	}
+`;
+
+const columnStyleBasedOnHasPageSkin = (hasPageSkin?: boolean) =>
+	hasPageSkin ? columnStyleWithPageSkin : columnStyle;
```
Comment (mxdvl) [inline dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx:215]: Not a blocker: we tend to do these checks inline in the `css` prop

File: dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx [hunk 3]
```diff
@@ -200,14 +200,30 @@ const columnStyle = css`
 	}
 `;
 
+const columnStyleWithPageSkin = css`
+	${columnStyle};
+	${from.leftCol}, ${from.wide} {
```
Comment (mxdvl) [inline dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx:205]: This is redundant, you only need the lowest of the two breakpoints, as `wide` is larger than `leftCol`.

File: dotcom-rendering/.vscode/settings.json (hunk 1 above)
Comment (deedeeh) [reply dotcom-rendering/.vscode/settings.json]: I haven't seen it in my commit so my bad! Addressed in aa0367810f0f3dada125612cb1c53b228f3bd821

File: dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx (hunk 2 above)
Comment (deedeeh) [reply dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx]: It took me a bit to understand what you mean but thanks to @DanielCliftonGuardian comment it was able to get it. Addressed in aa0367810f0f3dada125612cb1c53b228f3bd821

File: dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx (hunk 3 above)
Comment (deedeeh) [reply dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx]: Done aa0367810f0f3dada125612cb1c53b228f3bd821

File: dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx [hunk 4]
```diff
@@ -217,7 +233,10 @@ export const Column = ({
 	const ariaControls = `${columnId.toLowerCase()}Links`;
 
 	return (
-		<li css={[columnStyle, pillarDivider]} role="none">
+		<li
+			css={[columnStyleBasedOnHasPageSkin(hasPageSkin), pillarDivider]}
```
[the remark this answers is not in the document]
Comment (deedeeh) [reply dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx]: Thank you Daniel for the suggested change! Done aa0367810f0f3dada125612cb1c53b228f3bd821

**Review by mxdvl — APPROVED** [review]
As discussed offline, I think that it would be ideal to simplify the styles so we only apply `leftCol` and `wide` styles on fronts that do not have page skins. This will happen in a follow-up PR.

**Pull request merged**